solvexia_sdk/file/file.py

```diff
@@ -0,0 +1,75 @@
+#!/usr/bin/env python
+
+import requests
+import json
+import sys
+import math
+import os
+from fsplit.filesplit import Filesplit
+from solvexia_sdk import api
+
+class file:
+    def __init__(self, fileId):
+        self.fileId = fileId
+
+    def getFileMetadata(self):
+        response = api.apiGet(f"files/{self.fileId}/metadata")
+        api.statusCodeCheck(response, "Error getting metadata for file")
```

Comment:
It's useful to supply additional information in error messages like `fileId` in this case. This can help in debugging.

Reply:
Same applies to all other calls.

Reply:
fixed